XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,573 @@
+* The `lsfAccepted` flag is turned on in the credential.
+* The `Credential` object is moved from the issuer's owner directory to the subject's.
+
+## 7. Transaction: `CredentialDelete`
+
+This transaction deletes a `Credential` object.
+
+It can be executed by:
+* The issuer, anytime.
+* The account, anytime.
+* Anyone, after the expiration time is up.
+
+### 7.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`TransactionType`| ✔️|`string`|`UInt16`|The transaction type (`CredentialDelete`).|
+|`Account`| ✔️|`string`|`AccountID`|The transaction submitter.|
+|`Subject`| |`string`|`AccountID`|The person that the credential is for. If omitted, `Account` is assumed to be the subject.|
+|`Issuer`| |`string`|`AccountID`|The issuer of the credential. If omitted, `Account` is assumed to be the issuer.|
+|`CredentialType`|✔️|`string`|`Blob`|A (hex-encoded) value to identify the type of credential from the issuer.|
+
+### 7.2. Failure Conditions
+
+* The credential described by the `Subject`, `Issuer`, and `CredentialType` fields doesn't exist.
+* The `Account` isn't the issuer or account, and the expiration hasn't passed.
+
+### 7.3. State Changes
+
+If the transaction is successful:
+* The `Credential` object is deleted.
+
+## 8. Transactions: Common Fields
+
+### 8.1. Fields
+
+As a reference, [here](https://xrpl.org/docs/references/protocol/transactions/common-fields/) are the fields that all transactions currently have.
+
+<!--There are too many and I didn't want to list them all, it cluttered up the spec - but maybe it can be a collapsed section?-->
+
+| Field Name | Required? | JSON Type | Internal Type |
+|------------|-----------|-----------|---------------|
+|`CredentialIDs`| ✔️|`array`|`Vector256`|Credential(s) to attach to the transaction.|
+
+Transactions that accept this field:
+* Basically anything with a `Destination` field (like a payment) or something that has to be accepted by the destination (like an escrow/check).
+
+### 8.2. Failure Conditions
+
+* Any of the `CredentialIDs` isn't an object that exists.
+* Any of the `CredentialIDs` isn't a `Credential` object.
+* Any of the `CredentialIDs` is an expired `Credential` object.
+* `CredentialIDs` is included, but the destination doesn't have Deposit Authorization set up, or the transaction doesn't have a destination (e.g. `AccountSet`).
+* The `CredentialIDs` are not authorized by the destination.
+
+Note: the transaction will still fail if too many credentials are included. The exact list must be provided.
+
+### 8.3. State Changes
+
+* If the credential isn't valid, it gets blocked.
+* If the credential is expired, it is deleted.
+
+## 9. RPC: `deposit_authorized`
+
+The [`deposit_authorized` RPC method](https://xrpl.org/deposit_authorized.html) already exists on the XRPL. This proposal suggests some modifications to also support permissioned DEX domains.
```

Comment:
"permissioned DEX domains"? Is this a leftover from a different spec?

Reply:
Oops 😨